_posts/2019-08-11-pre-post-events-sustainable-train-travels.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title: "Pre-/Post Events: sustainable 🚄 train travels from and to RustFest Barcelona"
+authors:
+  - dns2utf8
+socialImageSrc: "/assets/posts/barcelona/train-map.png"
+---
```

Comment:
We should link to both the CfP and ticket sales in each blog post.

Reply:
Done